setup.py

```diff
@@ -25,8 +25,8 @@ def read(*parts):
 
 
 setup(
-    name='ttam-lintly',
-    version='0.7.6',
+    name='lintly23',
+    version='0.7.7',
```

Comment:
Not sure if you need to bump the version number (like... does it just treat it as an entirely new package?), but probably won't hurt either.